target/linux/mediatek/dts/mt7981b-cetron-ct3003.dts

```diff
@@ -45,13 +46,19 @@
 	leds {
 		compatible = "gpio-leds";
 
-		led_status_red: led_status_red {
+		led_status_red: led-0 {
 			label = "red:status";
```

Comment:
When the `label` property exists, `color` and `function` are ignored, so I guess these should be deleted whenever the label matches `$color:$function` (e.g., `red:status` will be auto-generated here).

Reply:
Yes. I notices if the label is removed then aliases section doesn't work. So you have any ideas how to avoid this?
```
	aliases {
		label-mac-device = &wan;

		led-boot = &led_status_blue;
		led-failsafe = &led_status_blue;
		led-running = &led_status_blue;
		led-upgrade = &led_status_blue;

		serial0 = &uart0;
	};
```

Reply:
This happens because of how get_dt_led() works.

https://github.com/openwrt/openwrt/blob/f909059b7473992e84d5701fa5021faa5bd76e67/package/base-files/files/lib/functions/leds.sh#L14

it only checks for the label and/chan property (and finally the node-name if all other fail).

Reply:
But once #13873 is merged...